libsupport/include/katana/ArrowVisitor.h

```diff
@@ -35,129 +22,166 @@ using is_list_type_patched = std::integral_constant<
               std::is_same<T, arrow::FixedSizeListType>::value>;
 template <typename T, typename R = void>
 using enable_if_list_type_patched =
-    arrow::enable_if_t<is_list_type_patched<T>::value, R>;
+    std::enable_if_t<is_list_type_patched<T>::value, R>;
```

Comment:
This can be removed with https://github.com/KatanaGraph/katana-enterprise/pull/1931